<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_sys_manage_outer_settings_label_win</class>
 <widget class="QDialog" name="w_sys_manage_outer_settings_label_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>打印机</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="lds_ComboBox" name="comboBox_printerName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>标签格式</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="lds_ComboBox" name="comboBox_labelFormat"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="lds_toolbar_button" name="pushButton_test">
       <property name="text">
        <string>测试</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_preview">
       <property name="text">
        <string>预览</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
